test/html/expect/test_package+ml/Markup/index.html

```diff
@@ -298,6 +301,30 @@ <h2 id="raw-html">
      </blockquote>
     </header>
    </section>
+   <section>
```

Comment:
These `section` elements look dubious to me but they are not related to the PR.

Reply:
Yeah, both `section` and `header` don't seem necessary. Each (section) heading starts a section, but I don't think that's needed.